### DISCUSSION TITLE
Cannot execute sinkListItem method

### DISCUSSION THREAD

**Discussion by Kilovatiy** [ueberdosis/tiptap#1966] [General Questions] [answered]
I like to have ability to sink/lift list items from a bubble menu, so I add handlers like this:
`sinkListItem() {
      this.editor.chain().focus().sinkListItem().run();
    },`
and 
`liftListItem() {
      debugger
      this.editor.chain().focus().liftListItem().run();
    },`
But it won't work. Can() method returns false as well. 
However, pressing 'Tab' works as expected. What needs to be done in order to execute these methods (lift and sink).

**Comment by philippkuehn** [comment]
hmm that should work. can you provide a codesandbox please?

**Reply by Kilovatiy** [reply]
I create an example [here](https://codesandbox.io/s/tiptap-and-a-bullet-list-5g6t4).
If I create bullet list and try to use a bubble menus,(it appears only on selected text), element 'sub-menu'  it wont sink an item, however if i set the cursor to the beginning of a li and press 'tab' all work fine. May it be that selection somehow make commands break?
Thank you for your reply!

**Comment by philippkuehn** [comment, marked as answer]
Instead of `this.editor.chain().focus().sinkListItem().run()` you have to call `this.editor.chain().focus().sinkListItem('listItem').run()`.

**Reply by Kilovatiy** [reply]
Thanks for the help!
